nistats/reporting.py

```diff
@@ -6,16 +6,193 @@
 
 import os
 import warnings
+from string import ascii_lowercase
+
 import numpy as np
+import pandas as pd
 from scipy import stats
 import nilearn.plotting  # overrides the backend on headless servers
+from nilearn.image.resampling import coord_transform
 import matplotlib
 import matplotlib.pyplot as plt
+from scipy import ndimage
 from patsy import DesignInfo
+
 from .design_matrix import check_design_matrix
 matplotlib.rc('xtick', labelsize=20)
 
 
+def _local_max(data, min_distance):
+    """Find all local maxima of the array, separated by at least min_distance.
+    From https://stackoverflow.com/a/22631583/2589328
+
+    Parameters
+    ----------
+    data : array_like
+        3D array of with masked values for cluster.
+
+    min_distance : :obj:`int`
+        Minimum distance between local maxima in ``data``, in terms of
+        voxels (not mm).
+
+    Returns
+    -------
+    xy : :obj:`numpy.ndarray`
```

Comment:
'xyz' would make more sense.

Reply:
You're right, `xy` is not good. I think I took it from the StackOverflow answer I used as the basis for the function. Is`ijk` alright though? I'd prefer to more clearly distinguish coordinates in mm (normally `xyz`) and matrix indices.